keyboards/rainkeebs/trailmix/readme.md

```diff
@@ -0,0 +1,27 @@
+# Trailmix
+
+![Trailmix](https://i.postimg.cc/wB4Zk1pS/IMG-20230326-222616.jpg)
+
+
+Ortho split 40% keyboard by rainkeenbs with encoders.
+
+* Keyboard Maintainer: [ATron789](https://github.com/atron789)
+* Hardware Supported: `Elite Pi`, `Pro Micro Type-C USB`, `Elite-C (v4)`, `nice!nano v2` controllers
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rainkeebs/trailmix:default
+
+Flashing example for this keyboard:
+
+    make rainkeebs/rainkeeb:default:flash
+(Flash one side of the keyboard at the time)
```

Comment:
I don't know if it's the norm or not, hence why I added this line of info

Reply:
Shouldn't be needed, in theory. The split docs should include, IIRC.

But also would need a space between the lines.